dask/dataframe/tests/test_dataframe.py

```diff
@@ -2438,33 +2454,60 @@ def test_info():
     ]
 
     for df in test_frames:
-        buf_pd, buf_da = StringIO(), StringIO()
-
         ddf = dd.from_pandas(df, npartitions=4)
-        df.info(buf=buf_pd)
-        ddf.info(buf=buf_da, verbose=True, memory_usage=True)
-
-        stdout_pd = buf_pd.getvalue()
-        stdout_da = buf_da.getvalue()
-        stdout_da = stdout_da.replace(str(type(ddf)), str(type(df)))
-
-        assert stdout_pd == stdout_da
+        _assert_info(df, ddf)
 
     buf = StringIO()
     ddf = dd.from_pandas(pd.DataFrame({'x': [1, 2, 3, 4], 'y': [1, 0, 1, 0]}, index=range(4)), npartitions=4)
 
     # Verbose=False
     ddf.info(buf=buf, verbose=False)
     assert buf.getvalue() == unicode("<class 'dask.dataframe.core.DataFrame'>\n"
-                                     "Data columns (total 2 columns):\n"
-                                     "x      int64\n"
-                                     "y      int64\n"
+                                     "Columns: 2 entries, x to y\n"
                                      "dtypes: int64(2)")
 
     # buf=None
     assert ddf.info(buf=None) is None
 
 
+def test_groupby_multilevel_info():
+    # GH 1844
+    from io import StringIO
+    from dask.compatibility import unicode
+
+    # TODO This should be fixed in pandas 0.18.2
+    if pd.__version__ == '0.18.0':
+        from pandas.core import format
+    else:
+        from pandas.formats import format
```

Comment:
Yeah, pandas < 0.19 support is going away. I hope to merge that in the next day or two (if all goes well). Could either keep the pandas < 0.19 support, and I'll remove it in that PR, or just wait until that's merged beforehand.